backend/account/tests/test_views.py

```diff
@@ -4,14 +4,15 @@
 
 
 class GetOrCreateAccountTests(TestCase):
-
     def setUp(self):
         self.factory = RequestFactory()
 
     def test_get_or_create_account(self):
-        request = self.factory.post('/get_or_create_account',
-                                    data='{"credential": "test_credential"}',
-                                    content_type='application/json')
+        request = self.factory.post(
```

Comment:
Not sure why this test passes with only `credential` being passed (and not `client_id`) 🤔

Reply:
Adjusting it and committing to see if tests succeed

Reply:
Test succeeds with client_id added here.